definitions/json/State.json

```diff
@@ -14,12 +14,20 @@
     "DisplayOrder": 2,
     "TitleDisplay": "# Case Number: ${ethosCaseReference} </br> <h4>${flagsImageAltText}</h4>"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "AWAITING_SUBMISSION_TO_HMCTS",
+    "Name": "AWAITING_SUBMISSION_TO_HMCTS",
```

Comment:
I think Name and Description should be
Awaiting Submission to HMCTS
https://tools.hmcts.net/confluence/display/CCE1D/Common+States+Model

Reply:
Incorporated.